AdminUi/test/AdminUi.Tests.Integration/API/BaseApi.cs

```diff
@@ -118,6 +122,13 @@ private async Task<HttpResponse<T>> ExecuteRequest<T>(HttpMethod method, string
             RawContent = responseRawContent
         };
 
+        if (hasCookies)
```

Comment:
I would probably inline `hasCookies`:

```suggestion
        if (httpResponse.Headers.TryGetValues("Set-Cookie", out var cookies))
```